src/main/kotlin/no/nav/familie/ef/mottak/mapper/ArkiverDokumentRequestMapper.kt

```diff
@@ -23,14 +23,12 @@ object ArkiverDokumentRequestMapper {
 
         val vedleggsdokumenter = SøknadTreeWalker.finnDokumenter(kontraktssøknad).map { tilDokument(it) }
 
-        // TODO legge til søknadsdokumentJson når integrasjoner takler flere variantfomater.
-        @Suppress("UNUSED_VARIABLE")
         val søknadsdokumentJson =
                 Dokument(soknad.søknadJson.toByteArray(), FilType.JSON, null, "hoveddokument", soknad.dokumenttype)
         val søknadsdokumentPdf =
                 Dokument(soknad.søknadPdf!!.bytes, FilType.PDFA, null, "hoveddokument", soknad.dokumenttype)
-        val dokumenter: List<Dokument> = listOf(søknadsdokumentPdf) + vedleggsdokumenter
-        return ArkiverDokumentRequest(soknad.fnr, false, dokumenter)
+        val hoveddokumentvarianter = listOf(søknadsdokumentPdf, søknadsdokumentJson)
+        return ArkiverDokumentRequest(soknad.fnr, false, hoveddokumentvarianter, vedleggsdokumenter)
```

Comment:
dette er flisespikk: parameterverdien `false` gjør det vanskelig å forstå hva som er parameter nummer to - vet ikke hva andre tenker - bør man "skrive ut" parameterlisten her eller la det være?
 
De tre andre er jo selvforklarende. 🤔